Websites/webkit.org/wp-content/themes/webkit/style.css

```diff
@@ -42,6 +42,7 @@ Version: 1.0
     --submit-button-text-color: hsl(0, 0%, 100%);
     
     --search-glyph: url('images/search.svg#dark');
+    --search-glyph-light: url('images/search.svg#light');
```

Comment:
should we maybe give this a different name (or even just hardcode it for the one use case) so it doesn't confuse future developers who wonder "why do we have this when `--search-glyph` has the same value in dark mode?

Reply:
`--search-glyph` behavior changes based on color scheme. `--search-glyph-light` stays light in both light and dark mode, which is exactly what is needed to specify in the unique desktop layout header case (it should always use "light"). 

I thought about defining it as `--search-glyph-inverse`, but given that there was no need to for different setting in dark mode, it would have no useful meaning. `--search-glyph-light` seems the most apt term for what this custom property is defined as.